owl/SOMA.owl

```diff
@@ -1192,6 +1213,15 @@ Such an event does NOT fall under the definition of Accident here. An example of
     
 
 
+    <!-- http://www.ease-crc.org/ont/SOMA.owl#MotionData -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA.owl#MotionData">
+        <rdfs:subClassOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#InformationObject"/>
+        <rdfs:comment>An information object containing data about the motion of some subject(s) and possibly also the motions of the objects they interact with, typically captured via motion tracking systems.</rdfs:comment>
```

Comment:
How is the object supposed to be linked to the actual data stored in some DB?

Reply:
This question could be asked of every InformationObject, most of which were not added by this PR. The answer is that there should be (at least) an InformationRealization attached to an InformationObject, and InformationRealizations have various data properties such as data format and persistent IRI to make sense of how to read them and where to find them.